docs/reference/snapshot-restore/apis/get-repo-api.asciidoc

```diff
@@ -0,0 +1,137 @@
+[[get-snapshot-repo-api]]
+=== Get snapshot repository API
+++++
+<titleabbrev>Get snapshot repository</titleabbrev>
+++++
+
+Gets information about one or more registered
+<<snapshots-register-repository,snapshot repositories>>.
+
+////
+[source,console]
+----
+PUT /_snapshot/my_repository
+{
+  "type": "fs",
+  "settings": {
+    "location": "my_backup_location"
+  }
+}
+----
+// TESTSETUP
+////
+
+[source,console]
+----
+GET /_snapshot/my_repository
+----
+
+[[get-snapshot-repo-api-request]]
+==== {api-request-title}
+
+`GET /_snapshot/<repository>`
+
+`GET /_snapshot`
+
+[[get-snapshot-repo-api-path-params]]
+==== {api-path-parms-title}
+
+`<repository>`::
+(Required, string)
+Comma-separated list of snapshot repository names used to limit the request.
+Wildcard (`*`) expressions are supported.
++
+To get information about all snapshot repositories registered in the
+cluster, omit this parameter or use `*` or `_all`.
+
+[[get-snapshot-repo-api-query-params]]
+==== {api-query-parms-title}
+
+`local`::
+(Optional, boolean) If `true`, the request gets information from the local node
+only. If `false`, the request gets information from the master node. Defaults to
+`false`.
+
+`master_timeout`::
+(Optional, <<time-units, time units>>) Specifies the period of time to wait for
+a connection to the master node. If no response is received before the timeout
+expires, the request fails and returns an error. Defaults to `30s`.
+
+`timeout`::
```

Comment:
I don't think we're using this parameter right now, let's not document it here

Reply:
Thanks for catching this.